packages/lexicon/package.json

```diff
@@ -24,12 +20,12 @@
     "directory": "packages/lexicon"
   },
   "dependencies": {
-    "@atproto/common-web": "*",
-    "@atproto/identifier": "*",
-    "@atproto/nsid": "*",
-    "@atproto/uri": "*",
+    "@atproto/common-web": "workspace:^",
+    "@atproto/identifier": "workspace:^",
+    "@atproto/nsid": "workspace:^",
+    "@atproto/uri": "workspace:^",
     "iso-datestring-validator": "^2.2.2",
-    "multiformats": "^9.6.4",
+    "multiformats": "^9.9.0",
```

Comment:
Multiformats [was resolved](https://github.com/bluesky-social/atproto/blob/bbc6a2c0fd7e2ecbafcb4006eeeb24c657a4a808/yarn.lock#L9330) to `9.9.0` anyway, so bumped it everywhere.